File: history/0.1.2/tatala-java/src/com/qileyuan/tatala/proxy/DefaultProxy.java
package com.qileyuan.tatala.proxy;

import org.apache.log4j.Logger;

import com.qileyuan.tatala.socket.to.TransferObject;

public class DefaultProxy {
	Logger log = Logger.getLogger(DefaultProxy.class);
	public Object execute(TransferObject to){
		log.error("This is DefaultProxy. You need extend it by specific proxy.");
		return null;
	}
}

File: history/0.1.2/tatala-client-csharp/src/QiLeYuan/Tatala/proxy/DefaultProxy.cs
using System;
using QiLeYuan.Tatala.socket.to;
using QiLeYuan.Tools.debug;

namespace QiLeYuan.Tatala.proxy {
    public class DefaultProxy {
        public virtual Object execute(TransferObject to) {
            Logging.LogError("This is DefaultProxy. You need extend it by specific proxy.");
            return null;
        }
    }
}
